src/poppler.mk

```diff
@@ -26,6 +26,8 @@ define $(PKG)_BUILD
         ./configure \
         $(MXE_CONFIGURE_OPTS) \
         --disable-silent-rules \
+        $(if $(BUILD_STATIC),--disable-shared --enable-static,\
+            --enable-shared --disable-static) \
```

Comment:
`$(MXE_CONFIGURE_OPTS)` sets `--disable-shared` etc. and these two lines aren't needed.